core/src/main/java/cucumber/runtime/formatter/JSONFormatter.java

```diff
@@ -379,4 +387,11 @@ private void addEmbeddingToHookMap(byte[] data, String mimeType) {
         }
         return resultMap;
     }
+    
+    private String getDateTimeFromTimeStamp(long timeStampMillis) {
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.XXX");
```

Comment:
This seems to result in dates that look like `1970-01-01T00:00:00.Z`. I think you meant to use `SSS` for milliseconds.

Reply:
Ah yes, you are right. I ll update this.